### DISCUSSION TITLE
Preventing fullstop dot ( . ) from triggering etc or \dots

### DISCUSSION THREAD

**Discussion by TryerGit** [James-Yu/LaTeX-Workshop#3648] [Q&A] [answered]
Hello,

Consider the animated gif below:

![latexworkshop](https://user-images.githubusercontent.com/70434658/212448736-ea7f8fc8-6e30-468d-b3f9-7907a93cd9a7.gif)

As of now, the default behaviour seems to be that a full stop automatically triggers / places focus on an inbuilt snippet for `\dots`. However, the usual / normal way of typing after a full stop, to start the next line/paragraph would just be to press <kbd>Enter</kbd>. Pressing <kbd>Enter</kbd>, unfortunately accepts the snippet suggestion for `\dots` which is not expected and is cumbersome.

Is there a way to disable only this particular snippet expansion for `.` -- the full stop?

Thank you.

**Comment by jlelong** [comment, marked as answer]
I cannot reproduce the behaviour you describe. On my configuration, typing a dot after at the end of a word does not trigger snippet suggestion. Can you reproduce this with all extensions but LW disabled? Could you also try with an empty `settings.json` file.

You may also be interested in setting `editor.acceptSuggestionOnEnter` to `false` to stop accepting suggestions on enter.

**Reply by TryerGit** [reply]
> You may also be interested in setting `editor.acceptSuggestionOnEnter` to `false` to stop accepting suggestions on enter.

Thank you. This worked perfectly. I set it up in workspace settings.json so as to make it specific for LaTeX .tex projects and not to affect what may be going on with the editor for other file types in other folders.

**Reply by jlelong** [reply]
Good news. Yet, I am interested in understanding where this behaviour is coming from. Can you reproduce it with all extensions but LW disabled?

**Reply by TryerGit** [reply]
I am on an older version of LW and VSCode as you may know. Yet, let me disable all else and update shortly.

**Reply by TryerGit** [reply]
@jlelong Your intuition was correct. With everything else but LW disabled, the problem does not occur. See image below:

![latexworkshop](https://user-images.githubusercontent.com/70434658/212460038-7139a779-6af8-423c-a640-d0f3ff717bb1.gif)

By  enabling the extensions one at a time incrementally, I have been able to confirm that the issue comes from HyperSnips extension. Let me investigate that elsewhere.
